.github/workflows/gx-plugin.yml

```diff
@@ -0,0 +1,93 @@
+name: GX Plugin
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - ".github/workflows/gx-plugin.yml"
+      - "metadata-ingestion-modules/gx-plugin/**"
+      - "metadata-ingestion/**"
+      - "metadata-models/**"
+  pull_request:
+    branches:
+      - master
+    paths:
+      - ".github/**"
+      - "metadata-ingestion-modules/gx-plugin/**"
+      - "metadata-ingestion/**"
+      - "metadata-models/**"
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  gx-plugin:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 3.0.3
+      DATAHUB_TELEMETRY_ENABLED: false
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.10"]
+        include:
+          - python-version: "3.8"
+            extraPythonRequirement: "great-expectations~=0.15.12"
+          - python-version: "3.8"
+            extraPythonRequirement: "great-expectations~=0.16.0 numpy~=1.26.0"
+          - python-version: "3.8"
+            extraPythonRequirement: "great-expectations~=0.17.0"
+          - python-version: "3.10"
+            extraPythonRequirement: "great-expectations~=0.15.12"
+          - python-version: "3.10"
+            extraPythonRequirement: "great-expectations~=0.16.0 numpy~=1.26.0"
```

Comment:
we can probably get away with only testing each version of GX with one python version. we do something similar with airflow

Reply:
makes sense